Project.toml

```diff
@@ -0,0 +1,12 @@
+name = "NMF"
+uuid = "6ef6ca0d-6ad7-5ff6-b225-e928bfa0a386"
+license = "MIT"
+version = "0.4.1"
+
+[deps]
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
Test should actually be a test dependency, which lives in its own section.
```toml
[extras]
Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"

[targets]
test = ["Test"]
```
Any other packages which are used in testing but not in the package should be under `[extras]` and similarly listed in the `test` target.